### DISCUSSION TITLE
How to get dashboard panel for plugin itself?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5343] [Ideas]
I need something like this:
![image](https://github.com/user-attachments/assets/49ab0919-c774-4c02-a750-f1b5c3d7d10c)

just for the plugin.   
I can imagine using admin links from plugin.xml...  or adding dashboard links there ...     Or using e_dashboard addon. 

It should be displayed on the main dashboard as the next panel.  Or in the plugin itself.    

Where to look to customize dashboard?  I did this long time ago with special template but I forgot how and I would like to have something more sophistic... 

You can have dashboard panel for group of plugins or just for one plugin with more admin options.
 
What do you think? Thanks

**Comment by tgtje** [comment]
@Jimmi08 in BS 3 admin theme has
`		$panelType = array(
			'core-infopanel' => 'panel-default',
			'admin_menu'     => '**panel-primary**',`

maybe something as panel-secondary ?? (plug-menu ;  ok theming too i gues)  just as idea

**Reply by Jimmi08** [reply]
It was pretty easy in fact (no change in core files, at least I don't remember them). You just need a new admin layout: 

![image](https://github.com/user-attachments/assets/1c3b2ea7-5116-4520-b60e-a899d2d036a8)

and use your own code to get what you want.  With e_dashboard new/any method you can output what you want.  

![image](https://github.com/user-attachments/assets/1d935718-083a-4693-adf0-cc89055296dd)
